src/layout/actions/workspace.rs

```diff
@@ -182,7 +182,7 @@ impl LayoutTree {
 
             // Get the root container of the next workspace
             let next_work_children = self.tree.children_of(next_work_ix);
-            if cfg!(debug_assertions) {
+            if cfg!(debug_assertions) || !cfg!(disable_debug) {
```

Comment:
Good call, I was dotting it all over the place and the logic can be difficult to read if there are a bunch of `cfg!`s clogging it up.